components/base/Collapsible.tsx

```diff
@@ -60,6 +65,8 @@ type Props = { triggerText: string; children: any };
 
 // Collapsible component
 function Collapsible({ triggerText, children }: Props) {
+  const [active, setActive] = useState(false);
```

Comment:
`true` `active` state refers to the Collapsible being open.
This helps manage keyboard clicks.